### DISCUSSION TITLE
All images are tagged with `uncommited` even though there is nothing uncommited.

### DISCUSSION THREAD

**Discussion by jadlr** [basecamp/kamal#378] [Q&A] [answered]
Could this have something to do with the `.env` file, that I'm creating dynamically on the CI?

How can I prevent this from happening?

Thanks!

**Comment by bdeak4** [comment, marked as answer]
you can fix this issue by adding `.env` file to `.gitignore`

**Reply by jadlr** [reply]
Perfect, thanks @bdeak4 :pray:

**Reply by RomanKrasavtsev** [reply]
In my case it was a file that was created during the previous CI pipeline step. So to debug it just print `git status` before `kamal deploy`
